lambdas/data-migration1/tests/test-collections.js

```diff
@@ -62,24 +73,39 @@ test.before(async () => {
 
   rulesModel = new Rule();
   await rulesModel.createTable();
+  await knexAdmin.raw(`create database "${testDbName}";`);
+  await knexAdmin.raw(`grant all privileges on database "${testDbName}" to "${testDbUser}"`);
+
+  // Run the migration against the newly created database
+  await handler({
+    env: {
+      KNEX_ASYNC_STACK_TRACES: 'true',
+      KNEX_DEBUG: 'true',
+      PG_HOST: knexConfig.connection.host,
+      PG_USER: knexConfig.connection.user,
+      PG_PASSWORD: knexConfig.connection.password,
+      PG_DATABASE: testDbName,
+    },
+  });
 });
 
-test.afterEach.always(async () => {
-  await knex('collections').truncate();
+test.afterEach.always(async (t) => {
+  await t.context.knex('collections').del();
```

Comment:
This is the critical/needed update, as you can't truncate a table that has FK constraints, etc.